server/utils/utils.go

```diff
@@ -65,3 +68,41 @@ func IsJiraCloudURL(jiraURL string) (bool, error) {
 	}
 	return strings.HasSuffix(u.Hostname(), ".atlassian.net"), nil
 }
+
+type JiraStatus struct {
+	State string `json:"state"`
+}
+
+// IsJiraAccessible checks if `/status` endpoint of the Jira URL is accessible
+// and responding with the correct state which is "RUNNING"
+func IsJiraAccessible(jiraURL string) (bool, error) {
+	u, err := url.Parse(jiraURL)
+	if err != nil {
+		return false, nil
+	}
+	if u.Host == "" {
+		return false, errors.Errorf("Invalid URL, no hostname: %q", jiraURL)
+	}
+
+	jURL := strings.TrimSuffix(u.String(), "/")
+	r, err := http.Get(jURL + "/status")
+	if err != nil {
+		return false, nil
+	}
+	if r.StatusCode != http.StatusOK {
+		return false, nil
+	}
+
+	resBody, err := ioutil.ReadAll(r.Body)
+	defer r.Body.Close()
+	if err != nil {
+		return false, nil
+	}
+	var j JiraStatus
+	err = json.Unmarshal(resBody, &j)
+	if err != nil {
+		return false, nil
+	}
+
+	return j.State == "RUNNING", nil
```

Comment:
The video the PR description is tested with Jira Cloud. Has this PR also been tested with a Jira Server instance? You can spin up a Jira Server instance on port `8080` by running `docker-compose up` in the root directory of this repo.

Reply:
Updated video.